carball/tests/analysis_test.py

```diff
@@ -4,7 +4,9 @@
 import shutil
 import subprocess
 import traceback
+from io import BytesIO
 
+from carball.analysis.utils.pandas_manager import PandasManager
```

Comment:
I think you should ctrl-alt-o and ctrl-alt-L more often. will fix imports ordering and spacing.